docs/source/Introduction.rst

```diff
@@ -1,2 +1,106 @@
+    <p><em>Thank you for using nginx.</em></p>
+    ...
+
+Building new images is great, but sharing our work with others let’s them review our work, critique how we built them, and offer improved versions. Our newly built Nginx image could be published at quay.io or docker.io to share it with the world. Everything needed to run the Nginx application is provided in the container image. Others could easily pull it down and use it, or make improvements to it.
+
+Standardizing on container images and `Container Registries`_ enable a new level of collaboration through simple consumption. This simple consumption model is possible because every major Container Engine and Registry Server uses the Open Containers Initiative (OCI_) format. This allows users to find, run, build, share and deploy containers anywhere they want. Podman and other `Container Engines`_ like CRI-O, Docker, or containerd can create and consume container images from docker.io, quay.io, an on premise registry or even one provided by a cloud provider. The OCI image format facilitates this ecosystem through a single standard.
+
+For example, if we wanted to share our newly built Nginx container image on quay.io it’s easy. First log in to quay::
+
+    podman login quay.io
+Input::
+
+    Username: USERNAME
+    Password: ********
+    Login Succeeded!
+
+Nex, tag the image so that we can push it into our user account::
+
+    podman tag localhost/nginx quay.io/USERNAME/nginx
+
+Finally, push the image::
+
+    podman push quay.io/USERNAME/nginx
+
+Output::
+
+    Getting image source signatures
+    Copying blob 38c40d6c2c85 done
+    Copying blob fee76a531659 done
+    Copying blob c2adabaecedb done
+    Copying config 7f3589c0b8 done
+    Writing manifest to image destination
+    Copying config 7f3589c0b8 done
+    Writing manifest to image destination
+    Storing signatures
+
+Notice that we pushed four layers to our registry and now it’s available for others to share. Take a quick look::
+
+    podman inspect quay.io/fatherlinux/nginx
+
+Output::
+
+    [
+        {
+            "Id": "7f3589c0b8849a9e1ff52ceb0fcea2390e2731db9d1a7358c2f5fad216a48263",
+            "Digest": "sha256:7822b5ba4c2eaabdd0ff3812277cfafa8a25527d1e234be028ed381a43ad5498",
+            "RepoTags": [
+                "quay.io/USERNAME/nginx:latest",
+    ...
+
+To summarize, Podman makes it easy to find, run, build and share containers.
+
+* Find: whether finding a container on dockerhub.io or quay.io, an internal registry server, or directly from a vendor a couple of find, or pull commands make it easy
+* Run: pre-built images make it easy to run entire applications, or start from a basic Linux base image based  on an existing Linux distribution
+* Build: creating new layers with small tweaks, or major overhauls is easy
+* Share: podman let’s you push your newly built containers anywhere you want with a single command
+
+For more deeper instructions on use cases, take a look at our `Tutorials` page.
```

Comment:
and maybe a link to the Commands page on read the docs?  http://docs.podman.io/en/latest/Commands.html

Reply:
I like that idea. Working on it.